app/controllers/likes_controller.rb

```diff
@@ -0,0 +1,7 @@
+class LikesController < ApplicationController
+  def index
+    puts params
+    Like.create(author: User.find_by(id: params['user_id']), post: Post.find_by(id: params['post_id']))
+    redirect_to "/users/#{params['user_id']}/posts/#{params['post_id']}"
+  end
```

Comment:
- [ ] Instead of `def index` we have to creake likes in Likes `def create` with POST method , for ex:
```ruby
def create
      @post = Post.find(params[:post_id])
      @like = @current_user.likes.new(post_id: @post.id)

      respond_to do |format|
        if @like.save
          format.html { redirect_to @post }
        else
          format.html { redirect_to @post }
        end
      end
    end
```